demo/vl-progress-bar-item.html

```diff
@@ -0,0 +1,77 @@
+                    &lt;vl-progress-bar-item&gt;Stap 1/3: Aanvraag&lt;/vl-progress-bar-item&gt;
+                    &lt;vl-progress-bar-item&gt;Stap 2/3: Gegevens&lt;/vl-progress-bar-item&gt;
+                    &lt;vl-progress-bar-item&gt;Stap 3/3: Bevestigen&lt;/vl-progress-bar-item&gt;
+                &lt;/vl-progress-bar&gt;
+    
+                &lt;vl-progress-bar data-vl-numeric&gt;
+                    &lt;vl-progress-bar-item data-vl-active&gt;Stap 1/3: Aanvraag&lt;/vl-progress-bar-item&gt;
+                    &lt;vl-progress-bar-item&gt;Stap 2/3: Gegevens&lt;/vl-progress-bar-item&gt;
+                    &lt;vl-progress-bar-item&gt;Stap 3/3: Bevestigen&lt;/vl-progress-bar-item&gt;
+                &lt;/vl-progress-bar&gt;
+    
+                &lt;vl-progress-bar data-vl-numeric&gt;
+                    &lt;vl-progress-bar-item&gt;Stap 1/3: Aanvraag&lt;/vl-progress-bar-item&gt;
+                    &lt;vl-progress-bar-item data-vl-active&gt;Stap 2/3: Gegevens&lt;/vl-progress-bar-item&gt;
+                    &lt;vl-progress-bar-item&gt;Stap 3/3: Bevestigen&lt;/vl-progress-bar-item&gt;
+                &lt;/vl-progress-bar&gt;
+    
+                &lt;vl-progress-bar data-vl-numeric&gt;
+                    &lt;vl-progress-bar-item&gt;Stap 1/3: Aanvraag&lt;/vl-progress-bar-item&gt;
+                    &lt;vl-progress-bar-item&gt;Stap 2/3: Gegevens&lt;/vl-progress-bar-item&gt;
+                    &lt;vl-progress-bar-item data-vl-active&gt;Stap 3/3: Bevestigen&lt;/vl-progress-bar-item&gt;
+                &lt;/vl-progress-bar&gt;
+            </code>
+        </pre>
+        <div class="demo">
+            <vl-progress-bar data-vl-numeric>
+                <vl-progress-bar-item>Stap 1/3: Aanvraag</vl-progress-bar-item>
+                <vl-progress-bar-item>Stap 2/3: Gegevens</vl-progress-bar-item>
+                <vl-progress-bar-item>Stap 3/3: Bevestigen</vl-progress-bar-item>
+            </vl-progress-bar>
+
+            <vl-progress-bar data-vl-numeric>
+                <vl-progress-bar-item data-vl-active>Stap 1/3: Aanvraag</vl-progress-bar-item>
+                <vl-progress-bar-item>Stap 2/3: Gegevens</vl-progress-bar-item>
+                <vl-progress-bar-item>Stap 3/3: Bevestigen</vl-progress-bar-item>
+            </vl-progress-bar>
+
+            <vl-progress-bar data-vl-numeric>
+                <vl-progress-bar-item>Stap 1/3: Aanvraag</vl-progress-bar-item>
+                <vl-progress-bar-item data-vl-active>Stap 2/3: Gegevens</vl-progress-bar-item>
+                <vl-progress-bar-item>Stap 3/3: Bevestigen</vl-progress-bar-item>
+            </vl-progress-bar>
+
+            <vl-progress-bar data-vl-numeric>
+                <vl-progress-bar-item>Stap 1/3: Aanvraag</vl-progress-bar-item>
+                <vl-progress-bar-item>Stap 2/3: Gegevens</vl-progress-bar-item>
+                <vl-progress-bar-item data-vl-active>Stap 3/3: Bevestigen</vl-progress-bar-item>
+            </vl-progress-bar>
```

Comment:
nodig om een aparte demo pagina voor item alleen te hebben?
en dan zeker om nog ne keer 4 verschillende te hebben?

Reply:
Initieel had ik geen gebruik van het data-vl-active attribuut op de vl-progress-bar pagina, nu toevallig wel bij de interactie demo's. Ik vind het niet slecht om voor elk attribuut een demo te voorzien, in dit geval dus ook voor de vl-progress-bar-item data-vl-active. Indien het je stoort dat het er 4 zijn, kan ik de eerste en laatste weg doen.

Reply:
en de 2e :-)